### PR TITLE
WLSSB new stuff and Fix formats

### PR DESCRIPTION
**Pull request by Lycanium-Z**
Added the newest staff
New Alfastorm and me
RIP HiroZ

Other tweaks when needed

### CODE REVIEW

**Comment by Lycanium-Z** [comment]
Tested and works!

File: mods/wlssb/random-teams.js [hunk 1]
```diff
@@ -347,6 +346,22 @@ class RandomSeasonalRegStaffTeams extends RandomTeams {
 				},
 				nature: 'Modest',
 			},
```
Comment (DeathlyPlays) [inline mods/wlssb/random-teams.js]: Mystifi is Admin again?

File: mods/wlssb/statuses.js [hunk 2]
```diff
@@ -99,17 +87,16 @@ exports.BattleStatuses = {
 	mystifi: {
 		exists: true,
 		onStart: function () {
-			this.add('c', '~Mystifi', '__I\'ll HM01 u faster then sanic m89__');
```
Comment (DeathlyPlays) [inline mods/wlssb/statuses.js]: Mystifi~

File: mods/wlssb/statuses.js [hunk 3]
```diff
@@ -186,32 +173,62 @@ exports.BattleStatuses = {
...
+				this.add('html|<button name="send" value="/forfeit">Hey ' + pokemon.side.name + '! Click me to win!</button>');
+			} else if (heckle === 1) {
+				this.add("raw|<div class=\"broadcast-red\"><b>The server is restarting soon.</b><br />Please finish your battles quickly. No new battles can be started until the server resets in a few minutes.</div>");
+			} else if (heckle === 2) {
+				this.add('c', pokemon.name, pokemon.side.name + ' sucks');
+			} else if (heckle === 3) {
+				this.add('-message', pokemon.name + ' fainted!');
+				this.add('-message', 'Just Kidding!');
+			} else if (heckle === 4) {
+				this.add('j|~Totally an Admin');
```
Comment (DeathlyPlays) [inline mods/wlssb/statuses.js]: Wouldn't these be |j|, |l|, |c|

Comment (Lycanium-Z) [reply mods/wlssb/statuses.js]: No

File: mods/wlssb/random-teams.js (hunk 1 above)
Comment (Lycanium-Z) [reply mods/wlssb/random-teams.js]: %Lycanium Z: Wait Mystifi is admin again?
~HoeenHero: that was temp
%Lycanium Z: Oh

File: mods/wlssb/statuses.js (hunk 2 above)
Comment (Lycanium-Z) [reply mods/wlssb/statuses.js]: %Lycanium Z: Wait Mystifi is admin again?
~HoeenHero: that was temp
%Lycanium Z: Oh

**Comment by Volco** [comment]
@Lycanium-Z everythings tested? and ready?

**Comment by Lycanium-Z** [comment]
Almost(TM) Just gotta test the fixing of formats

**Comment by Volco** [comment]
okay lmk when it is fully fixed and tested. then i'll merge it

**Comment by Lycanium-Z** [comment]
Ok done and tested and ready for merge!

**Pull request merged**